geth/txqueue/txqueue_manager_test.go

```diff
@@ -137,9 +137,9 @@ func (s *TxQueueTestSuite) TestCompleteTransactionMultipleTimes() {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			_, err := txQueueManager.CompleteTransaction(tx.ID, TestConfig.Account1.Password)
+			_, errCompleteTransaction := txQueueManager.CompleteTransaction(tx.ID, TestConfig.Account1.Password)
```

Comment:
Why so long?

Reply:
Why not? It's obvious name.

Reply:
Yes but why so long? Why not just `e`? Usually, the longer a variable name, the broader its scope.

Reply:
It could be e. But.. Why not?) In that case what should be next name of error after e? ee?
errCompleteTransaction is obvious name. It's simple. You don't need to comment it.

Reply:
There won't be another case. If there's, the name will be just longer. Anyway, it's just a matter or taste, I prefer longer variables for broader scope and shorter for narrower. Following and knowing this rule just makes you understand existing code faster.

Reply:
Is this rule fixed in our docs? Should we check "a matter or taste" on code review?

Reply:
Sorry?
Docs is not what's doled out to us, this is what we ourselves create during open discussions. If there's something you follow which makes code better, why shouldn't you share it?
I'm not asking to fix that based on my matter of taste, by that I'm saying my comment isn't relevant any more as the matter isn't worth the efforts discussing it.

Reply:
i'll try to describe better my point.
As far as i know we are going to make oun community. If so we need strict CR rules on which all major contributors agreed. If we all agred that "less scope = shorter name", this should be fixed in doc to be project standard for core developers and all new contributers.

Reply:
I agree but your question was strange given that we don't have such a document yet at all and I never asked to fix anything due to just my taste.
And not all comments are major, some of them are just for discussion like this one and written along with major comments. Or they're just some questions.